tests/views/adScreen-test.js

```diff
@@ -270,13 +271,44 @@ describe('AdScreen', function() {
       mutingForAutoplay: true
     };
 
-    var wrapper = Enzyme.mount(
+    const wrapper = Enzyme.mount(
       <AdScreen
         controller={mockController}
         skinConfig={defaultSkinConfig}
       />);
-    var unmuteIcons = wrapper.find(UnmuteIcon);
+    const unmuteIcons = wrapper.find(UnmuteIcon);
     expect(unmuteIcons.length).toBe(0);
   });
 
+  it('should hide controlBar after resize', () => {
+    let controlBarVisible = true;
+
+    mockController.startHideControlBarTimer = () => {
+      controlBarVisible = false;
+    };
+
+    expect(controlBarVisible).toBe(true);
+
+    const node = document.createElement('div');
+    Enzyme.mount(
+      <AdScreen
+        controller={mockController}
+        skinConfig={mockSkinConfig}
+        componentWidth={400}
+        fullscreen={false}
+      />, node
+    );
+
+    Enzyme.mount(
```

Comment:
As per documentation, it unmounts component and mounts it again, so you don't simulate resize here.
Please use `setProps` to update width

Reply:
\+